docs/guides/src/main/getting-started/getting-started-openshift.adoc

```diff
@@ -9,43 +9,43 @@ summary="Get started with Keycloak on OpenShift">
 
 == Before you start
 
-Install https://code-ready.github.io/crc/[Red Hat Code Ready Containers] and follow the steps in the documentation to install a 
+. Install https://code-ready.github.io/crc/[Red Hat Code Ready Containers] and follow the steps in the documentation to install a 
 local OpenShift cluster.
 
-Make sure the cluster is functional by executing the following command:
-
+. Make sure the cluster is functional by entering the following command:
++
 [source,bash,subs="attributes+"]
 ----
 crc status
 ----
 
-If everything is OK you should see an output similar to this:
-
+. Look for output similar to the following to confirm the cluster is working.
++
 [source,subs="attributes+"]
 ----
 CRC VM:          Running
 OpenShift:       Running
 ...
 ----
 
-Log in as the user `developer`:
-
+. Log in as the user `developer`:
++
 [source,bash,subs="attributes+"]
 ----
 oc login -u developer -p developer
 ----
 
-For this guide, we are going to create a new project called `keycloak`. For that, execute the following command:
+. Create a project called `keycloak` by entering the following command:
 
 [source,bash,subs="attributes+"]
```

Comment:
The bash script should relate to the enumeration item, therefore add a "+" to keep them together.

```suggestion
. Create a project called `keycloak` by entering the following command:
+
[source,bash,subs="attributes+"]
```